sw/make-deb

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# --------------------------------------------------------------------------
+# Create deb-package after compile
+#
+# Author: Bernhard Bablok
+# License: GPL3
+#
+# Website: https://github.com/tomek-szczesny/pistackmon
+#
+# --------------------------------------------------------------------------
+
+PLATFORM="$1"
+if [ -z "$PLATFORM" ]; then
+  echo "usage: $0 platform" >&2
+  exit
+else
+  # strip some stuff added by makefile and convert to lower-case
+  PLATFORM="${PLATFORM##*_}"
+  PLATFORM="${PLATFORM,,}"
+fi
+
+echo "pistackmond daemon" > description-pak
+
+sudo checkinstall \
+  --default \
+  --install=no \
+  --pkgname=pistackmond-$PLATFORM \
+  --pkgversion=1.0.0 \
+  --pkgrelease=1 \
+  --pkglicense=GPL3 \
+  --pkgsource=https://github.com/tomek-szczesny/pistackmon \
+  --pkgaltsource=https://github.com/tomek-szczesny/pistackmon \
+  --maintainer=t.szczesny@10g.pl \
```

Comment:
Is this a common practice to include e-mail address in crawler-friendly format? I'd rather use my customary "t.szczesny at 10g in Poland". Unless it is a custom then I won't oppose :)

Reply:
Or perhaps you should be the maintainer of deb package? Not entirely sure about the importance of this, given that pistackmond is not likely to join Ubuntu repos anytime soon. :)

Reply:
I will change this to my address. It is already visible in too many places.